### PR TITLE
change logic of discount

### PR DESCRIPTION
**Pull request by MaltsevaKsenia**
we don't have discount table any more, now discount is one of the tour entity field

### CODE REVIEW

File: src/main/java/com/epam/controller/ToursController.java [hunk 1]
```diff
@@ -104,7 +105,7 @@ public ModelAndView addTour(@RequestParam String tourType, @RequestParam String
                     .pricePerUnit(addPricePerPerson)
                     .hotelId(1) //stub
                     .description(tourDescription)
-                    .discountId(1) //stub
+                    .discountId(addDiscount)
```
Comment (goponenkoev) [inline src/main/java/com/epam/controller/ToursController.java]: Is it still the name `discountId`? Maybe would better rename

File: src/main/java/com/epam/controller/ToursController.java [hunk 2]
```diff
@@ -104,7 +105,7 @@ public ModelAndView addTour(@RequestParam String tourType, @RequestParam String
                     .pricePerUnit(addPricePerPerson)
                     .hotelId(1) //stub
```
Comment (goponenkoev) [inline src/main/java/com/epam/controller/ToursController.java:111]: Why hotelId is still stubbed?

Comment (ghost) [reply src/main/java/com/epam/controller/ToursController.java:111]: I believe we can fix it in another request, this is only about discount, isnt it?

**Review by ghost — APPROVED** [review]

File: src/main/java/com/epam/controller/ToursController.java (hunk 2 above)
Comment (MaltsevaKsenia) [reply src/main/java/com/epam/controller/ToursController.java:111]: yes, only about discount

File: src/main/java/com/epam/controller/ToursController.java (hunk 1 above)
Comment (MaltsevaKsenia) [reply src/main/java/com/epam/controller/ToursController.java]: done

**Review by goponenkoev — APPROVED** [review]

**Review by Cisfinitum — APPROVED** [review]

**Review by VNTee — APPROVED** [review]

**Pull request merged**